dimod/binary/binary_quadratic_model.py

```diff
@@ -847,10 +852,18 @@ def add_linear_from_array(self, linear: Sequence):
     """Alias for :meth:`add_linear_from`."""
 
     def add_offset(self, bias):
-        """Add offset to to the model."""
+        """Add offset to to the model.
+
+        .. deprecated:: 0.10.0
+
+            ``BinaryQuadraticModel.add_offset()`` will be removed in dimod 0.12.0.
+            Use ``bqm.offset += bias`` instead.
```

Comment:
I think given the context, this will be clear. This is not really intended for new users, it's only useful to people who are already using the method in which case I think we can rely on them making the leap.